syft/formats/common/spdxhelpers/document_namespace.go

```diff
@@ -44,7 +43,7 @@ func DocumentNamespace(name string, src source.Description) string {
 	u := url.URL{
 		Scheme: "https",
 		Host:   "anchore.com",
-		Path:   path.Join(internal.ApplicationName, identifier),
+		Path:   path.Join(src.Name, identifier),
```

Comment:
why use src.Name here? (seems like a non 1:1 changee)

Reply:
This isn't a 1:1 change, but if a different application is running, we probably shouldn't indicate it's _syft_, but also we should probably have a different way to ascertain a URL

Reply:
This should not have been _source_.Name but rather _sbom.descriptor_.Name